Classes/Classes/miniphonedir.cpp

```diff
@@ -0,0 +1,101 @@
+#include<iostream> 
+
+using namespace std; 
+
+  
+
+class UserData { 
+
+private: 
+
+    string name; 
+
+    string last_name; 
+
+    string email; 
+
+    string phone_number; 
+
+    /*Method*/ 
+
+public: 
+
+    void set_name(string name_temp); 
+
+    string get_name(); 
+
+    
+
+     
+
+}; 
+
+  
+
+void UserData::set_name(string name_temp) 
+
+{ 
+
+    name = name_temp; 
+
+} 
+
+  
+
+string UserData::get_name() 
+
+{ 
+
+    return name; 
+
+} 
+
+  
+
+int main(void) 
+
+{ 
+
+    int option; 
+
+    string name; 
+
+    do { 
+
+        cout << "Directorio Telefonico\n"; 
+
+        cout << "Seleccione una opcion:\n\n"; 
+
+        cout << "1. Agregar Nuevo\n\n"; 
+
+        cout << "Seleccione:"; 
+
+        cin >> option; 
+
+  
+
+        switch (option) 
+
+        { 
+
+        case 1:  
+
+            system("cls"); 
+
+            UserData User_New; 
```

Comment:
Once this case is executed the memory will go out of scope, we need to allocated dynamic memory using:
UserData* newUserPtr = new UserData();

In order to keep track of all contacts we can create a collection to store these pointers, e.g.:
std::list<UserData*> userDataList;
userDataList.insert(newUserPtr);

Reply:
A list object is exactly like a list in python?